package.json

```diff
@@ -86,6 +86,7 @@
     "@types/varint": "6.0.0",
     "@typescript-eslint/eslint-plugin": "5.37.0",
     "@typescript-eslint/parser": "5.37.0",
+    "array-from-async": "^3.0.0",
```

Comment:
nit: sharing you past sentiment (I think), is it possible to avoid adding a new dependency for use in once place?

Reply:
this is a prollyfill for the [array-from-async](https://github.com/tc39/proposal-array-from-async) TC39 proposal, so i wouldnt necessarily call it the same kind of dependency as the others

but yes that's a fair point i shouldnt be a hypocrite :P

Reply:
oh tho actually this is a `devDependency`, so i personally dont have as much of a concern about it since its not part of shipping code

but ill still remove it :)